arrays&objects/studentDirectory.js

```diff
@@ -1,3 +1,6 @@
+//Information for each student is stored in an object
+// From those objects, functions pull information about age,email addresses and programming languages.
+// stores them in arrays and outputs them  
```

Comment:
I think this is showing in this PR because the [other PR](https://github.com/NosiphoManyase/TechUpSchool/pull/1#pullrequestreview-847802884) hasn't been merged yet :)